scopus/__init__.py

```diff
@@ -12,3 +12,4 @@
 from scopus.scopus_author import *
 from scopus.scopus_reports import *
 from scopus.scopus_search import *
+from scopus.scopus_author_search import *
```

Comment:
In Scopus 1.0 a new naming convention will apply, which can already be seen from the CitationOverview class.  The name of the class will follow the name of the Scopus API and the script name (and name of folder for cached files) will correspond to the last parts in the resource URL.  That is, in Scopus 1.0 I will rename your new class to AuthorSearch and the script to search_author.py.  But it would be much better if you implemented this already here!